addons/account/models/account_payment.py

```diff
@@ -447,6 +447,17 @@ def _get_method_codes_using_bank_account(self):
     def _get_method_codes_needing_bank_account(self):
         return []
 
+    def action_open_business_doc(self):
+        return {
+            'name': _("Payment"),
+            'type': 'ir.actions.act_window',
+            'view_mode': 'form',
+            'views': [(False, 'form')],
+            'res_model': 'account.payment',
+            'res_id': self.id,
+            'target': 'current',
```

Comment:
```suggestion
```
Default is current according to [this](https://www.odoo.com/documentation/16.0/developer/reference/backend/actions.html?highlight=view_mode#window-actions-ir-actions-act-window)